unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -82,9 +82,37 @@
       <entry name="top" value="2"/>
       <entry name="overlay" value="3"/>
     </enum>
+
+    <!-- Version 3 additions -->
+
+    <event name="new_output" since="3">
+      <description summary="notify of a new wl_output">
+        This event indicates that a new output has been created. If the client
+        wishes to create a new layer surface in response to this new output,
+        it must make an ack_new_output request with the provided serial
```

Comment:
If the client doesn't want to create a new layer surface, it still needs to send the ack.